swarm/network/protocol_test.go

```diff
@@ -209,14 +220,15 @@ func TestBzzHandshakeNetworkIDMismatch(t *testing.T) {
 }
 
 func TestBzzHandshakeVersionMismatch(t *testing.T) {
+	lightNode := false
 	addr := RandomAddr()
-	s := newBzzHandshakeTester(t, 1, addr)
+	s := newBzzHandshakeTester(t, 1, addr, lightNode)
 	id := s.IDs[0]
 
 	err := s.testHandshake(
-		correctBzzHandshake(addr),
-		&HandshakeMsg{Version: 0, NetworkID: 3, Addr: NewAddrFromNodeID(id)},
```

Comment:
I have mixed feelings about changing the hardcoded values to extracted variables. The idea is - if someone changes the `DefaultNetworkID` in the code, a test should fail, because this is changing the protocol and the contract. Peers would no longer be able to communicate.

You must know very well what you are doing when changing this, and I am afraid that hacking the actual value we tests against (in this case `DefaultNetworkID` and `Version`) has the opposite effect - tests will automatically pass, and you will notice that you changed the contract only once you deploy this.

Reply:
I created a new const called `TestProtocolNetworkID` and used that in tests. I understand now that why using the DefaultNetworkID was a bad idea, however generally it's a good practice to not used hardcoded constants in the test code (or for that matter in any code), because it may lead to errors in unrelated tests when you change the constant in NetworkID tests.

What do you think?

Reply:
I am happy with `TestProtocolNetworkID`